templates/vpc.yaml

```diff
@@ -502,6 +502,31 @@ Resources:
           Value: !Ref Project
         - Key: "OwnerEmail"
           Value: !Ref OwnerEmail
+  QuarantineSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: Security Group for bastians
```

Comment:
is this accurate, shouldn't it be about quarentining?

Reply:
yeah, updated